<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WasteManagement</class>
 <widget class="QMainWindow" name="WasteManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paper Waste</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../home/src/res.qrc">
    <normaloff>:/images/batfile.png</normaloff>:/images/batfile.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpStart">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpEnd">
        <property name="displayFormat">
         <string>dd MMM yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdRefresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CustomQueryView" name="trvWaste"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Entry"/>
    <addaction name="actionEdit_Entry"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Entry">
   <property name="text">
    <string>New Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEdit_Entry">
   <property name="text">
    <string>Edit Entry</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../home/src/res.qrc"/>
  <include location="../../home/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
